packages/audio-block/src/AudioBlock.spec.ct.tsx

```diff
@@ -133,4 +135,19 @@ describe('Audio Block', () => {
         mount(<AudioBlockWithStubs />);
         cy.get(AudioBlockSelector).should('have.css', 'flex-direction', 'column');
     });
+
+    it('renders an audio block without download button as its set in the security settings', () => {
```

Comment:
```suggestion
    it('renders an audio block with download button as its set in the security settings', () => {
```

Reply:
Can you also add a test for not rendering it?

Reply:
yep, added